graphics/rsvg_frame.cpp

```diff
@@ -54,7 +53,9 @@ static RawFrame *render_from_rsvg_handle(RsvgHandle *rsvg) {
 
 static void rsvg_tryinit( ) {
     if (!rsvg_is_init) {
+        #ifndef GLIB_VERSION_2_36
```

Comment:
did g_type_init go away in this version? is it possible to make this a comparison (if version is less than 2.36 then g_type_init())?

(or is GLIB_VERSION_2_36 defined for 2.36 or later?)

Reply:
It's done automatically (?) now.

> GTypeDebugFlags has been deprecated since version 2.36 and should not be used in newly-written code.
>
> g_type_init() is now done automatically

Source: https://developer.gnome.org/gobject/stable/gobject-Type-Information.html

I followed some guidance I found online about wrapping this in a preprocessor macro: https://lists.ubuntu.com/archives/ubuntu-devel/2013-January/036332.html